### PR TITLE
Upgrade liquibase to 3.5.x version

### PR DESCRIPTION
**Pull request by georgejunior**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-4986

## Checklist: I completed these to help reviewers :)

- [x ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [ x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by mozzy11** [comment]
@georgejunior  have you seen the travis Falure

**Comment by georgejunior** [comment]
> @georgejunior have you seen the travis Falure

yes but am failing to reproduce the bug

File: pom.xml [hunk 1]
```diff
@@ -278,7 +278,7 @@
 			<dependency>
 				<groupId>org.liquibase</groupId>
 				<artifactId>liquibase-core</artifactId>
-				<version>2.0.5</version>
+				<version>3.6.3</version>
```
Comment (ODORA0) [inline pom.xml]: @georgejunior You have got two Pull requests upgrading liquibase, I think you should close one. For this, you could compare versions and their dependencies, relate them to what's failing as seen in the failure logs. You can check this out https://mvnrepository.com/artifact/org.liquibase/liquibase-core/2.0.5 to help

Comment (georgejunior) [reply pom.xml]: let me close one of them

Comment (gitcliff) [reply pom.xml]: @georgejunior are you able to build successful with "mvn clean install" on your machine?

Comment (georgejunior) [reply pom.xml]: no i wasn't able to

Comment (ODORA0) [reply pom.xml]: @georgejunior have you tried with later versions than 3.6.3 ofcourse older than the current 2.0.5?

Comment (gitcliff) [reply pom.xml]: @georgejunior when you are not able to compile successfully with mvn clean install on your machine then automatically you will get travis failure, so you need to concentrate first on why your getting build failure on your machine by looking at your error logs in the terminal or you can share here the pastebin link

Comment (georgejunior) [reply pom.xml]: @gitcliff here is the log https://pastebin.com/5FgixtZ6

Comment (gitcliff) [reply pom.xml]: @georgejunior according to your error log in regards to "Cannot find symbol error" you can have alook here https://stackoverflow.com/questions/25706216/what-does-a-cannot-find-symbol-compilation-error-mean
Also critically analyze your ClassLoaderFileOpener.java class and the methods it overrides

**Pull request closed without merging**